Rakefile

```diff
@@ -11,7 +11,7 @@ end
 task :connection do
   require "active_record"
   require_relative "spec/support/connection_config"
-  ActiveRecord::Base.establish_connection(db_config.except(:database))
+  ActiveRecord::Base.establish_connection(db_config.merge(database: 'template1'))
```

Comment:
I'm excluding the database from the config here because it might not exist yet. I guarantee this will fail CI

Reply:
At least for me, on a clean postgres install its unable to connect because no database is specified 'template1' is a database thats defined by postgres and should definely be there on any install.

Reply:
Oh interesting, TIL about `template1`. Somehow I have never heard of it before.

Reply:
I've been opening connections without a db for the whole time without problems:
```
~/P/activerecord-pg_enum master ×
$ export TEST_DATABASE=pg_enum_test
~/P/activerecord-pg_enum master ×
$ psql pg_enum_test
psql: FATAL:  database "pg_enum_test" does not exist
~/P/activerecord-pg_enum master ×
$ be appraisal 6.1 bundle exec rake spec:setup
>> BUNDLE_GEMFILE=/Users/adam/Projects/activerecord-pg_enum/gemfiles/6.1.gemfile bundle exec rake spec:setup
~/P/activerecord-pg_enum master ×
$ psql pg_enum_test
Expanded display is used automatically.
Null display is "¤".
psql (10.10, server 9.6.15)
Type "help" for help.

pg_enum_test=# \q
```

Can I get some repro steps for this, or even just more information about your DB configuration?